pennylane/expval/__init__.py

```diff
@@ -50,5 +50,44 @@
     expval/cv
 """
 
-from .qubit import *
-from .cv import *
+from pennylane.qnode import QNode, QuantumFunctionError
+
+from .qubit import * #pylint: disable=unused-wildcard-import,wildcard-import
+from .cv import * #pylint: disable=unused-wildcard-import,wildcard-import
+
+from .cv import __all__ as _cv__all__
+from .qubit import __all__ as _qubit__all__
+
+class Identity(object): #pylint: disable=too-few-public-methods,function-redefined
```

Comment:
```suggestion
class Identity: #pylint: disable=too-few-public-methods,function-redefined
```
In Python 3, all classes inherit from `object` implicitly by default, so does not need to be written. The exception is when you write code targeting both Python 2 and Python 3, as Python 2 did not have this behaviour.

Reply:
Old habits...